app/main/views/index.py

```diff
@@ -70,9 +70,25 @@ def resources(**kwargs):
 @main.route('/whats-on')
 @setup_subscription_form
 def whats_on(**kwargs):
+    articles = api_client.get_articles_summary()
+    index = randint(0, len(articles) - 1)
+
+    future_events = api_client.get_events_in_future(approved_only=True)
+    past_events = []
+    if len(past_events) < 3:
+        all_past_events = api_client.get_events_past_year()
```

Comment:
Should probably update the API to include a call to get back the last n events in the past, but for now its probably ok.